File: 1791-richest-customer-wealth/1791-richest-customer-wealth.cs
public class Solution {
    public int MaximumWealth(int[][] accounts) {
        int maxWealth = 0;
        foreach (var customer in accounts) {
            int wealth = customer.Sum();
            maxWealth = Math.Max(maxWealth, wealth);
        }
        return maxWealth;
    }
}

File: 1791-richest-customer-wealth/1791-richest-customer-wealth.js
var maximumWealth = function(accounts) {
    let maxWealth = 0;
    for (let i = 0; i < accounts.length; i++) {
        let wealth = accounts[i].reduce((acc, val) => acc + val, 0);
        maxWealth = Math.max(maxWealth, wealth);
    }
    return maxWealth;
};